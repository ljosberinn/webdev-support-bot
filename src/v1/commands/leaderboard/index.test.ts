import leaderboardHandler from '.';

test('retuns with sample leaderboard message', async () => {
  const msg: any = {
    channel: { send: jest.fn() },
    delete: jest.fn(),
  };

  await leaderboardHandler(msg, 0);

  expect(msg.channel.send.mock.calls[0]).toMatchSnapshot();
});
