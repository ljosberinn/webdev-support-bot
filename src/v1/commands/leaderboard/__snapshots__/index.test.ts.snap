// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`retuns with sample leaderboard message 1`] = `
Array [
  Object {
    "embed": Object {
      "author": null,
      "color": 16670549,
      "description": "No users found.",
      "fields": Array [],
      "footer": Object {
        "iconURL": "https://github.com/ljosberinn/webdev-support-bot/blob/master/logo.png?raw=true",
        "text": "Leaderboard: Helpful Users",
      },
      "title": "Leaderboard: Helpful Users",
      "url": null,
    },
  },
]
`;
